src/vorta/store/models.py

```diff
@@ -79,7 +79,7 @@ class BackupProfileModel(BaseModel):
     ssh_key = pw.CharField(default=None, null=True)
     compression = pw.CharField(default='lz4')
     exclude_patterns = pw.TextField(null=True)
-    exclude_if_present = pw.TextField(null=True)
+    exclude_if_present = pw.TextField(null=True, default="[] .nobackup\n[] .vortaignore")
```

Comment:
`.nobackup` is fine, but I wouldn't introduce a new standard, like `.vortaignore`. Why not add `CACHEDIR.TAG`, which is an existing [standard](https://bford.info/cachedir/) to support the related Borg flag.

Reply:
Wasn't aware about CACHEDIR.TAG, added it now.